libs/langgraph/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -162,6 +174,19 @@ def _should_bind_tools(model: LanguageModelLike, tools: Sequence[BaseTool]) -> b
     return False
 
 
+def _get_model(model: LanguageModelLike) -> BaseChatModel:
```

Comment:
Consider `_unwrap_chat_model` or `_extract_chat_model` or something similar that reflects the purpose of the function.